File: week_10/day_4/Athlete.java
public class Athlete {
  private int totalDistance;
  private AthleteLog log;

  public Athlete(AthleteLog log){
       this.log = log;
  
  }

  public void run(int distance){
    totalDistance += distance;
    this.log.distance(distance, totalDistance);
  }
}

File: week_11/day_2/water_bottle.js/athlete.js

var athlete = {
  hydration: 100,
  distance: 0,
  run: function(distance){
    for (var i = 0; i < distance; i++){
    if (this.hydration !== 0){
    this.hydration -= 10;
    this.distance  += 1;
  }
 } 
},
  drink: function(waterBottle){
    this.hydration += waterBottle.drink();
  }
}


module.exports = athlete;